datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +200,115 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self) -> Iterable[Tuple[str, str]]:
+
+        url = self.config.get_sql_alchemy_url()
+        engine = create_engine(url, **self.config.options)
+
+        with engine.connect() as conn:
```

Comment:
I think it makes sense that your functions accept a `conn` connection instead of building one. Normally you don't want to create new connections frequently, instead you make a single connection pool `conn` and pass that around. I would do that in the `get_workunits`, build the `conn` and then pass it to the `_get_view_lineage_workunits` and any other functions that need it.